src/main/java/uk/gov/ons/ctp/response/collection/exercise/schedule/SchedulerConfiguration.java

```diff
@@ -129,7 +129,9 @@ private static EventDTO getEventDTOFromJobKey(final Scheduler scheduler, final J
 
       return result;
     } catch (SchedulerException e) {
-      log.error("Failed to get event information for job with key {} - {}", jobKey, e.getMessage());
+      log.with("jobKey", jobKey)
+          .with("message", e.getMessage())
+          .error("Failed to get event information for job");
```

Comment:
when we're logging an exception, we should put the exception as the final parameter in the method call to either `.warn` or `.error` and let the logging framework log the cause and the stack trace etc.